hbase-assembly/src/main/assembly/client.xml

```diff
@@ -104,24 +56,44 @@
 
   <dependencySets>
     <dependencySet>
-      <outputDirectory>lib/shaded-clients</outputDirectory>
-      <includes>
-        <include>org.apache.hbase:hbase-shaded-client</include>
-        <include>org.apache.hbase:hbase-shaded-mapreduce</include>
-        <include>org.apache.hbase:hbase-shaded-client-byo-hadoop</include>
-      </includes>
+      <outputDirectory>lib</outputDirectory>
+      <useTransitiveDependencies>true</useTransitiveDependencies>
```

Comment:
nit:  <!-- Exclude artifacts added in the sub-directories to avoid duplication -->

Reply:
Synced to the other xml